server/database_setup.py

```diff
@@ -36,38 +88,40 @@
                 image_url=record['imageUrl'],
             )
 
-            for stat in record['stats']:
-                item_stat = ModelItemStats(
-                    stat=stat['stat'],
-                    min_value=stat['minStat'],
-                    max_value=stat['maxStat']
-                )
-                base.db_session.add(item_stat)
-                base.db_session.commit()

```

Comment:
I removed this `commit()`, I think that the stat object shouldn't be added if something else in the item fails.